docs/REST API/rest-api.md

```diff
@@ -6253,7 +6257,6 @@ List software that can or will be automatically installed during macOS setup. If
         "name": "FirefoxInsall.pkg",
         "platform": "darwin",
         "version": "125.6",
-        "self_service": true,
```

Comment:
Dev note: Dropping this because for setup experience self-service doesn't matter and label scoping rules don't apply.